source/common/upstream/outlier_detection_impl.h

```diff
@@ -189,6 +248,11 @@ class DetectorConfig {
   uint64_t enforcingConsecutive5xx() { return enforcing_consecutive_5xx_; }
   uint64_t enforcingConsecutiveGatewayFailure() { return enforcing_consecutive_gateway_failure_; }
   uint64_t enforcingSuccessRate() { return enforcing_success_rate_; }
+  uint64_t consecutiveLocalOriginFailure() { return consecutive_local_origin_failure_; }
```

Comment:
I might be missing something obvious since it isn't on any of the other existing methods here, but this is a `const` method, right?